test/text-parser.test.js

```diff
@@ -9,7 +9,7 @@ test('Testing text-parser', async () => {
   expect(result).toBe('Hello World');
 });
 
-test.skip('Testing text-parser with new line', async () => {
+test('Testing text-parser with new line', async () => {
```

Comment:
Can you add a test with a document fragment vs. a whole document, to mimic what we'll do with a post?

Reply:
Can you explain a little bit?. I don't quite understand

Reply:
All your tests have `<!DOCTYPE html>....` and are full documents.  But what we mostly get from blog posts is `<p>some random snippet of html ... </p>` (i.e., not a full document).